src/main/java/it/gov/pagopa/reward/controller/InstrumentApiController.java

```diff
@@ -0,0 +1,19 @@
+package it.gov.pagopa.reward.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+@RequestMapping("/paymentinstrument")
+public interface InstrumentApiController {
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{userId}/{initiativeId}")
+    Mono<Void> cancelInstruments(@PathVariable("userId") String userId,
+                                 @PathVariable("initiativeId") String initiativeId);
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @PostMapping("/{userId}/{initiativeId}")
+    Mono<Void> rollbackInstruments(@PathVariable("userId") String userId,
```

Comment:
ok, this is a rollback for the cancel operation...
But a more generic name could be "activeInstruments"

Moreove the path of the REST API is ambigous...

POST /paymentinstrument/userId/initiativeId
just by reading this, we are not able to understang what's happening...

Maybe it's better something like

PUT /paymentinstrument/userId/initiativeId/active